src/routes/download/index.ts

```diff
@@ -477,4 +478,56 @@ router.get('/file/:form/:blob', async (req, res) => {
   });
 });
 
+router.post('/export/pdf/', async (req, res) => {
+  try {
+    const authToken: any = req.headers.authorization.split('Bearer ')[1];
+    const accessTokenId: any = req.headers.access_token;
+
+    const browser = await puppeteer.launch({
+      executablePath: process.env.CHROME_BIN || null,
```

Comment:
Have you found anything that could enable to run the chromium in another docker container, instead of the main back end one ?
So Split the responsabilités of the containers ?